joomla/joomla-cms
# [4.0] Improve Text on Modal: Batch process language labels

IMHO some of the text in the **Batch process the selected articles** modal is too long:
- "To Move or Copy your selection please select a Category." 
- "Do you want to move the items or make a copy of them?" 

This PR rewrites that text to:
- "Select a Category for your Selection"
- "Copy or Move the items?"

### Summary of Changes
On the current modal some of the text seems a bit too long:
![batch2](https://user-images.githubusercontent.com/1217850/57234833-d453bc80-7021-11e9-8912-672953a124f0.png)

### Testing Instructions
Content > Articles > select some articles > click on Batch
Select a category so that the Copy / Move question is shown.

### Expected result
This PR makes two of the labels shorter so that IMHO it looks nicer,
but also easier to read and understand:

![batch3](https://user-images.githubusercontent.com/1217850/57234830-d453bc80-7021-11e9-8d02-e6bc848f4843.png)

### Suggestion (NOT in this patch)
If this PR gets accepted then I suggest to move the radio buttons of Copy / Move to the left

![batch4-suggestion](https://user-images.githubusercontent.com/1217850/57234829-d3bb2600-7021-11e9-8453-44aa3a57e65f.png)

by removing the left margin in administrator/templates/atum/css/template.css line 470:
```CSS
.control-group .controls {
    margin-left: 220px;
}
```
